doc/news/_preparation_next_release.md

```diff
@@ -211,6 +244,9 @@ Thanks to Michael Zronek and Vanessa Kos.
 - The Markdown Shell Recorder now also tests if a command prints nothing to `stdout` if you add the check `#>`. *(René Schwaiger)*
 - We fixed some problems in the [Markdown Shell Recorder](https://master.libelektra.org/tests/shell/shell_recorder/tutorial_wrapper) test
   of [`kdb ls`](https://master.libelektra.org/doc/help/kdb-ls.md). *(René Schwaiger)*
+- The Markdown Shell Recorder test for [`kdb find`](https://master.libelektra.org/doc/help/kdb-find.md) now removes the configuration file
+  at the end of the test. *(René Schwaiger)*
+- We removed the broken auto unmounting feature from the [Markdown Shell Recorder][]. *(René Schwaiger)*
```

Comment:
What is the behavior now?

Reply:
The Markdown Shell Recorder does not do any auto unmounting. That behavior did not change. I just removed dead code.